doc/std/vision.md

```diff
@@ -32,23 +33,23 @@ call for contributions below.
 
 ### Standard library outline
 
-1. Core types and operations  
```

Comment:
I wonder if these whitespace fixes can be split into a separate PR, to make this review more focused.

Reply:
I don't think that's necessary.